homework8/ErikaTatchyn/task4.js

```diff
@@ -0,0 +1,14 @@
+// Write a function which accepts any number of arguments and returns a string in format "argument [index]: [argument]; argument [index]: [argument];", e.g."argument 0: 1; argument 1: 4; argument 2: 79;". More vivid description:
+function argumentsLogger(...args) {
+  let arr = [];
```

Comment:
`let` -> `const`
For that case u only add soem new values for array, but not rewrite.